views/list-role-band-levels.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Roles</title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+    </head>
+    <body>
+        <header class="bg-dark text-light">
+            <div class="container">
+                <div class="row">
+                    <div class="col-md-12">
+                        <h1>Roles</h1>
+                    </div>
+                </div>
+            </div>
+        </header>
+        <main class="container mt-4">
+            <section id="rolebandlevels" class="mt-4">
+                <h2>All roles' band levels</h2>
+                {% if rolebandlevels and rolebandlevels.length > 0 %}
+                <table class="table">
+                    <thead>
+                        <tr>
+                            <th scope="col">Role Title</th>
+                            <th scope="col">Band Level</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                    {% for RoleBandResponse in rolebandlevels %}
```

Comment:
What you think of using role here?

```suggestion
                    {% for role in rolebandlevels %}
```